src/fswAlgorithms/imageProcessing/centerOfBrightness/_UnitTest/test_centerOfBrightness.py

```diff
@@ -0,0 +1,186 @@
+
```

Comment:
:large_blue_circle: I have some thoughts on how we can improve the simplify our testing approach. It centers around no longer misusing the parameterize capability as a means to record what the test did.  I'll work an example and offer it in a PR after this PR.

Reply:
I'll be on the lookout!